web/src/compose_closed_ui.js

```diff
@@ -125,10 +125,7 @@ export function update_buttons_for_private() {
         !narrow_state.pm_ids_string() ||
         people.user_can_direct_message(narrow_state.pm_ids_string())
     ) {
-        $("#new_conversation_button").attr(
-            "data-tooltip-template-id",
-            "new_direct_message_button_tooltip_template",
-        );
+        $("#new_conversation_button").attr("data-conversation-type", "dm");
```

Comment:
Let's use `direct` to store this -- that's what we use in the APIs to distinguish the two message types.